storages/storages.go

```diff
@@ -61,10 +61,11 @@ type Storages struct {
 
 // Backend represents any storage in akubra cluster
 type Backend struct {
-	http.RoundTripper
-	Endpoint    url.URL
-	Name        string
-	Maintenance bool
+	Transports   transport.Container
+	RoundTripper http.RoundTripper
```

Comment:
Why do we still have a seperate RoundTripper, when we already have `Transports` with multiple RoundTrippers?